### DISCUSSION TITLE
Filament in Production?

### DISCUSSION THREAD

**Discussion by fpolli** [filamentphp/filament#14197] [General]
Is anyone using Filament in production on a high-volume site? How does it perform? I'm starting to learn it for my site, but I see references to poor performance and that makes me worry about committing my project to it. Other than that it looks great and I would love to work with it.

**Comment by e2d2-dev** [comment]
in my opinion performance just relies on your code. i have around 1000 customers and it's doing a great job.

**Comment by hizmarck** [comment]
In my experience, depends on the complexity of your interface, right now I have a problem with a simple resource view with 50 rows per page, bc I have like 7 action buttons that have some rules based on the role of the person, I guess that each time that the view is rebuilt is heavy for the view, which is sad. 

I'm reading about Livewire to build my interface differently, like splitting into different resources to the same Entity, idk I'm only figuring out how to resolve this.

**Comment by e2d2-dev** [comment]
#designIssues

A list with > 10-20 entries is just for bored(?) users | employees.

-> Create filters for resources
-> Put Widgets on the dashboard, which only pop up when a certain number is hit and link them to the filters

i got over 200 Resources on almost 200 models right now and everything is fine, even with over 10 actions on a model per view.
